build-tools/packages/build-cli/docs/update.md

```diff
@@ -0,0 +1,29 @@
+`flub update`
+=============
+
+Rewrite imports for Fluid Framework APIs to use the correct subpath import (/alpha, /beta. etc.)
+
+* [`flub update fluid-imports`](#flub-update-fluid-imports)
+
+## `flub update fluid-imports`
+
+Rewrite imports for Fluid Framework APIs to use the correct subpath import (/alpha, /beta. etc.)
+
+```
+USAGE
+  $ flub update fluid-imports [-v | --quiet] [--organize] [--onlyInternal]
+
+FLAGS
+  --onlyInternal  Use /internal for all non-public APIs instead of /alpha or /beta.
+  --organize      Organize the imports in any file that is modified. Note that this can make it more difficult to see
+                  the rewritten import changes.
+
```

Comment:
Is there an existing tool we can run on all of the files to do this in a pre-pass? Something we can enforce in our repo?

Reply:
> Is there an existing tool we can run on all of the files to do this in a pre-pass?

Not an existing tool - just this one. :) But yes, we can do a pre-pass across the repo like I did with aqueduct.

> Something we can enforce in our repo?

Once this is integrated into the client release group, we can create a task that runs it against all projects (see the `check:versions` task for example). We have a task in CI that checks for unexpectedly modified files, so if the task modifies any files, it'll fail CI. That will get us bare-bones protection at the cost of running this every build.

It could also be a part of policy, but policy is repo-wide and operates at the file level, so I'm not convinced it's the best place.